feature/profile/src/main/java/com/iteneum/profile/presentation/ProfileView.kt

```diff
@@ -28,18 +29,30 @@ import androidx.compose.ui.res.painterResource
 import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.text.font.FontWeight
 import androidx.compose.ui.unit.sp
+import androidx.hilt.navigation.compose.hiltViewModel
 import com.iteneum.designsystem.components.LpFloatingActionIconButton
 import com.iteneum.designsystem.components.LpOutlinedButton
 import com.iteneum.designsystem.theme.LeasePertTheme
 import com.iteneum.designsystem.R as R_DS
 import com.iteneum.profile.R
+import com.iteneum.profile.data.ProfileViewModel
 
 @Composable
 fun ProfileView(
-    /*TODO expected parameters as navigation and view model*/
+    navigation: () -> Unit /* TODO - ProfileView - Check navigation parameter*/ ,
```

Comment:
rename this parameter, it depends what do you want, if it's to navigate back you can rename it as "BackToDashBoard" or if it's to navigate to other view you can rename "navigateToNameView"

Reply:
It is for going back, by pressing the arrow component, but not implemented yet. Changed the name for the one you mentioned. Thank you!